src/test/codegen/external-no-mangle-fns.rs

```diff
@@ -14,32 +14,35 @@
 #![crate_type = "lib"]
 #![no_std]
 
+// Avoid functions being merged through calls to this dummy
```

Comment:
Likewise (`-C no-prepopulate-passes`).

Reply:
I've changed the other test to use `no-prepopulate-passes`, but not sure on this one, as it explicitly specifies `-O`.

Reply:
I went ahead and changed this one as well, as I really don't see a reason for it to use `-O`.

Reply:
Oh, I missed the `-O`, but I agree it doesn't seem relevant to the test. Still, cc @alexcrichton in case they remember why they wrote the test that way.

Reply:
Hm I'm not sure! Me being lazy with the copy/paste is the most likely reason, seems fine to switch it now!